package.json

```diff
@@ -39,11 +39,10 @@
   },
   "homepage": "https://github.com/energywebfoundation/passport-did-auth#readme",
   "dependencies": {
-    "@energyweb/credential-governance": "2.2.1-alpha.313.0",
-    "@energyweb/onchain-claims": "2.2.1-alpha.313.0",
-    "@energyweb/vc-verification": "2.2.1-alpha.313.0",
-    "@ensdomains/ens-contracts": "0.0.8",
-    "@ethersproject/providers": "5.4.4",
```

Comment:
Uninstalled `"@ethersproject/providers"` because we already have `ethers` installed